qemu/tests/cfg/blockdev_commit_with_ignore.cfg

```diff
@@ -0,0 +1,50 @@
+- blockdev_commit_with_ignore:
+    type = blockdev_commit_with_ignore
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 2G
+    snapshot_tags_data = sn1
+
+    image_size_sn1 = 2G
+    image_format_sn1 = qcow2
+    image_name_sn1 = sn1
+
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+
+    tmpdir = /home/tmp
+    mount_dir = /mnt/test
+    loopdev = ${tmpdir}/loopdev
+    backend_base = ${tmpdir}/disk.img
+    image_name_data = ${mount_dir}/data
+    vgname = test
+    lvname = testlvm1
+    pre_command = "mkdir -p ${tmpdir} && losetup -f > ${loopdev} &&"
+    pre_command += "qemu-img create -f raw ${backend_base} 4G &&"
+    pre_command += "losetup `cat ${loopdev}` ${backend_base} &&"
+    pre_command += "pvcreate `cat ${loopdev}` &&"
+    pre_command += "vgcreate ${vgname} `cat ${loopdev}` &&"
+    pre_command += "lvcreate -L 100M -n ${lvname} ${vgname} &&"
+    pre_command += "mkfs -t ext3 /dev/${vgname}/${lvname} && mkdir -p ${mount_dir} &&"
+    pre_command += "mount /dev/${vgname}/${lvname} ${mount_dir}"
+
+    post_command += "umount ${mount_dir} &&"
+    post_command += "lvremove -f /dev/${vgname}/${lvname} &&"
+    post_command += "vgremove -f ${vgname} &&"
```

Comment:
fyi, we can use lvm.EmulatedLVM, e.g. 
self._lvm = EmulatedLVM(params, get_data_dir())
self._lvm.setup()
self._lvm.cleanup()

cfg:
     vg_name = vg_test
     lv_name = lv_test
     lv_size = 1G
     block_raw_name = lvm_block       
     emulated_image = images/${block_raw_name}
     emulated_image_size = ${image_size_data1}
     force_remove_image = yes
     remove_emulated_image = yes

But it's OK for me about your implementation.